src/css/flags/time-slider-above.less

```diff
@@ -27,6 +21,12 @@
       }
   }
 
+  &.jw-state-idle:not(.jw-flag-cast-available), &.jw-state-error {
+    .jw-display {
+      padding: 0;
+    }
+  }
+
```

Comment:
Moved to increase specificity for the existing case and for error case. Ensures that the error display icon stays centered